firebase-migration-guide.md

```diff
@@ -55,23 +59,31 @@ Firebase Analytics requests in React Native Firebase are almost the same - mostl
 
 > Note: `logEvent` is asynchronous, so be sure to use `await` in front of the function call if you are using it in an `async` function.
 
-### Create a development build
+## Create a development build
 
 If you have been using Expo Go for development and testing, at this point, you will need to migrate to a [development build](https://docs.expo.dev/development/introduction/), as Expo Go does not include the native code used in React Native Firebase.
 
-[Follow these instructions for creating a development build](https://docs.expo.dev/development/create-development-builds/) and then install the build on your test device or simulator.
+1. Install `expo-dev-client` in your project by running the following command:
+
+```shell
+npx expo install expo-dev-client
+```
+
+2. [Follow these instructions from React Native Firebase library docs](https://rnfirebase.io/#managed-workflow) on how to add the config plugin(s) and provide paths to Google's service JSON files in the Expo config file.
```

Comment:
@kbrandwijk, here is the first part.

Reply:
> @kbrandwijk, here is the first part.

Oh right, I read 'follow these instructions' so I thought this was just the existing part moved around. My bad...